.github/workflows/test-pr.yaml

```diff
@@ -67,35 +67,12 @@ jobs:
       - name: "Generating tests for '${{ matrix.templates }}'"
         run: echo "${{ matrix.templates }}"
 
-      - name: Configure template
+      - name: "Configuring template with default options"
+        env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           set -e
-          cd src/${{ matrix.templates }}
-
-          # Configure templates only if `devcontainer-template.json` contains the `options` property.
-          optionProp=( $(jq -r '.options' devcontainer-template.json) )
-
-          if [ "${optionProp}" != "" ] && [ "${optionProp}" != "null" ] ; then  
-            options=( $(jq -r '.options | keys[]' devcontainer-template.json) )
-
-            if [ "${options[0]}" != "" ] && [ "${options[0]}" != "null" ] ; then
-              echo "(!) Configuring template options within 'src/${{ matrix.templates }}' folder"
-              for option in "${options[@]}"
-              do
-                option_key="\${templateOption:$option}"
-                option_value=$(jq -r ".options | .${option} | .default" devcontainer-template.json)
-
-                if [ "${option_value}" = "" ] || [ "${option_value}" = "null" ] ; then
-                  echo "Template '${{ matrix.templates }}' is missing a default value for option '${option}'"
-                  exit 1
-                fi
-
-                echo "(!) Replacing '${option_key}' with '${option_value}'"
-                option_value_escaped=$(sed -e 's/[]\/$*.^[]/\\&/g' <<<"${option_value}")
-                find ./ -type f -print0 | xargs -0 sed -i "s/${option_key}/${option_value_escaped}/g"
-              done
-            fi
-          fi
+          devcontainer templates apply --workspace-folder "src/${{ matrix.templates }}" --template-id "ghcr.io/${{ github.repository }}/${{ matrix.templates }}:latest"
```

Comment:
@joshspicer  Is there a bug when we don't pass template args?
Looks like the CLI is returning incorrect list of changed files 🤔 

![image](https://user-images.githubusercontent.com/24955913/199607961-e2aa5bfc-665b-40a7-a562-85184b92a724.png)

Reply:
It replaces correctly, the only issue with the returned `files[]`

Reply:
Oh wait. It also looks like the cmd is not just replacing the arguments, but it is also replacing the contents of the files (fetched from OCI artifact vs what is provided by the `workspace-folder`).

Hmm. I think that was how it was expected for `Add dev Container Config Cmd`. But in this case we can't use the cmd without tweaking or adding a new flag